sensor/common/clusterentities/store.go

```diff
@@ -54,18 +54,32 @@ type Store struct {
 	publicIPsListeners map[PublicIPsListener]struct{}
 
 	mutex sync.RWMutex
+
+	// entitiesMemorySize defines how many ticks old endpoint data should be remembered after removal request
+	// Set to 0 to disable memory
+	entitiesMemorySize  uint16
+	historicalEndpoints map[string]map[net.NumericEndpoint]*entityStatus
+	historicalIPs       map[net.IPAddress]map[string]*entityStatus
+	historyMutex        sync.RWMutex
 }
 
 // NewStore creates and returns a new store instance.
 // Note: Generally, you probably do not want to call this function, but use the singleton instance returned by
```

Comment:
Then why have this function? Where is it called and why?

Reply:
This is how the code was before my PR. I think this comment can be removed - it is up to us whether we want to use it as singleton or not.